website/lib/homepage/home_page.dart

```diff
@@ -0,0 +1,118 @@
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:website/homepage/featured_editor.dart';
+import 'package:website/homepage/inside_the_toolbox.dart';
+
+import '../breakpoints.dart';
+import 'call_to_action.dart';
+import 'editor_video_showcase.dart';
+import 'features.dart';
+import 'footer.dart';
+import 'header.dart';
+
+class HomePage extends StatefulWidget {
+  const HomePage();
+
+  @override
+  _HomePageState createState() => _HomePageState();
+}
+
+class _HomePageState extends State<HomePage> {
+  @override
+  Widget build(BuildContext context) {
+    final isSingleColumnLayout = Breakpoints.collapsedNavigation(context);
+
+    return Scaffold(
+      // TODO: move magic number to named constant
+      backgroundColor: const Color(0xFF003F51),
+      body: DrawerLayout(
+        isNavigationCollapsed: isSingleColumnLayout,
+        child: DefaultTextStyle.merge(
+          style: const TextStyle(
+            fontWeight: FontWeight.w400,
+            fontSize: 18,
+            height: 27 / 18, // TODO: why is this division happening here?
```

Comment:
It comes from the Figma prototype, where the line height was expressed this way. 

When implementing designs, I usually start with the exact values from the prototype and once everything is finalized, I go through these again and simplify them. I didn't have a chance to do that yet this time.

(that's also why there are some magic numbers and values like `1113` that don't seem to conform to any grid)

```suggestion
            height: 1.5,
```